PyTado/interface/api/base_tado.py

```diff
@@ -393,3 +397,48 @@ def get_running_times(self, date=datetime.datetime.now().strftime("%Y-%m-%d")) -
         request.params = {"from": date}
 
         return self._http.request(request)
+
+    def get_boiler_install_state(self, bridge_id: str, auth_key: str):
+        """
+        Get the boiler wiring installation state from home by bridge endpoint
+        """
+
+        request = TadoRequest()
+        request.action = Action.GET
+        request.domain = Domain.HOME_BY_BRIDGE
+        request.device = bridge_id
+        request.command = "boilerWiringInstallationState"
+        request.params = {"authKey": auth_key}
+
+        return self._http.request(request)
```

Comment:
are you sure the boiler stuff is supported by Tado X too? afaik it's currently only supported by v3/v2/... systems which means it should stay in my_tado.py

Reply:
I do not have a boiler, so I do not know. The action can't effectively used by TadoX bridges because they do not have an authKey!

I will make it abstract and move it to my_tado.py